rustiflow/src/flows/custom_flow.rs

```diff
@@ -39,36 +56,41 @@ impl Flow for CustomFlow {
                 ts_date,
             ),
             // Add here the initialization of the additional features.
+            inter_arrival_time_total: 0.0,
+            last_timestamp: Some(ts_date),
```

Comment:
Can we not reuse last timestamp of BasicFlow?

Reply:
Jup indeed, I will fix that!